build-4.9.sh

```diff
@@ -186,6 +199,7 @@ main()
   prepare_kernel
   build_kernel
   # build_libs
+  # NOTE(NK): Why is this commented out
```

Comment:
See the note above, but basically I have to figure out how to get post-build steps to make the `/dev` entries with dpkg. So I wanted to leave the code here for the future when I get it figured out.